user_manual/files/deleted_file_management.rst

```diff
@@ -53,7 +52,8 @@ by setting the ``trashbin_retention_obligation`` value. Files older than the
 ``trashbin_retention_obligation`` value will be deleted permanently.
 Additionally, Nextcloud calculates the maximum available space every time a new
 file is added. If the deleted files exceed the new maximum allowed space
-Nextcloud will expire old deleted files until the limit is met once again.
+Nextcloud will permanently delete old removed files which will expire until the
+limit is met once again.
```

Comment:
```suggestion
Nextcloud will permanently delete those trashed files with the soonest expiration until the
space limit is met again.
```

Reply:
The changes have been made. Please approval. Thanks 😉.